pkg/asset/manifests/openshift.go

```diff
@@ -135,6 +133,7 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 			Data:     data,
 		})
 	}
+	o.FileList = append(o.FileList, worker.Files()...)
```

Comment:
This makes me jumpy about reloading/regenerating these when they're loaded from disk during a multi-stage install.  Can we move `Worker` out from under `Openshift` into it's own writeable asset (like we did for masters in  7a396d9137, #1211)?  CC @staebler

Reply:
> This makes me jumpy about reloading/regenerating these when they're loaded from disk during a multi-stage install. Can we move `Worker` out from under `Openshift` into it's own writeable asset (like we did for masters in [7a396d9](https://github.com/openshift/installer/commit/7a396d9137140e9824becbe217392197b8c9529c), #1211)? CC @staebler

master's we use as input to `cluster` asset and that's why it makes sense to split that out in #1211 that worker machinesets are opaque to creating infra resources for now and I don't think we would need to do something like #1211 for the time being.

Reply:
> ... worker machinesets are opaque to creating infra resources for now...

So should we stop writing them at all?  Drop ghem from `Openshift` without making them writable assets? This is getting into #1119.

Reply:
> So should we stop writing them at all?

why, people can edit the manifest as a file. doesn't mean we don't write them :confused:

Reply:
> why, people can edit the manifest as a file. doesn't mean we don't write them confused

Because then we won't have structured info about chosen worker availability zones and such.  I'll work up some test-cases tonight.

Reply:
whatever we decide on that ^^ , I don't think it should be part of this PR.